### DISCUSSION TITLE
Experience Page Improvements - Tagging System

### DISCUSSION THREAD

**Discussion by ghost** [alan-turing-institute/AutSPACEs#371] [Ideas]
Provide a more interactive drop down tagging system components that shows the types of tags the user can choice from. 

Include location tags including: Transport, Hospitals, School, Transport for London, COVID etc. Combine use of titles and icons to represent tubes, buses, hospitals etc - ref to PECS & Mulberry Set. 

**User Feedback Labels:**
Experience Page
Tagging System

**Comment by katoss** [comment]
# Original user testing feedback

From https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/user-tests/sprint-6-feedback/summary-sprint6-feedback.md

**Tags**

- A C d: “It would be nice if there was some form part that was like ‘transport’, ‘hospitals’, ‘school’, whatever it was, and people would go in and get advice from users. But I don’t know if that is the intention. You probably create the data and then it goes off into a report and then people at London transport will access it when it is finished.”
- A C d: Would like a tagging system to tag categories like “transport”, “hospitals” etc.
- A (1) d: tags that would be useful: transport, covid (at the moment), other, not applicable
- A (1) d: “A word or phrase [referring to tags to allow more options] – drop down selection with explanation of what it means.”
- A C d: “Just the fact that, obviously, this only being a flat process of putting stuff in, I would like at some stage for someone to sign it, so some of the experiences can be going into thematic categories. [..] So that everything that is related to transport or health, for hospital environments is extracted, so we can actually go in and look at that.”
- A C d: Would like symbols for e.g. tubes, busses or hospitals as a support when picking tags

- → On the PECS symbols used on the ‘non-verbal’ site (https://c6q95y.axshare.com/#g=1&p=3-add-event-non-verbal-home-final): A C d: criticises ‘shop’ symbol (no text on sign) and expressions (hard to understand, “I think even emojis are better than that”)

**→ Tagging systems were not thematized in all user tests, because they were originally not part of the session’s schedule. However, when thematized, participants were positive about the idea.**
